### ISSUE TITLE
Unable to convergence

### ISSUE THREAD

**Issue by handsomefengfeng** [xw-hu/Mask-ShadowGAN#9]
I've trained the model on the given USR dataset.

None parameters were changed except the dataroot.

However, the generator was unable to convergence.

![image](https://user-images.githubusercontent.com/48850881/79034614-e6eb5400-7be9-11ea-83eb-f842fb9f9149.png)

Meanwhile, the discriminator convergence better than the generator.
![image](https://user-images.githubusercontent.com/48850881/79034630-113d1180-7bea-11ea-8daf-d79bac703648.png)

**Comment by handsomefengfeng** [comment]
Could you please provide any solution or suggestion?  Thanks!!!

**Comment by 1214710638** [comment]
> Could you please provide any solution or suggestion? Thanks!!!

I have the same problem, have you solved it yet?

**Comment by xw-hu** [comment]
How about re-starting training? Maybe the initial random seed is not good.

**Comment by handsomefengfeng** [comment]
> How about re-starting training? Maybe the initial random seed is not good.

The above is the result I re-trained on the provided USR dataset without any configs changed in the train_MaskShadowGAN.py.

You mean it's the unsatisfactory of initial random seed that make the loss not so smooth.
Could you please tell how to make the initial random seed appropriate?

Thanks~~!   :)

**Comment by GqqGGqqG** [comment]
Yeah, I didn't change the parameters too.However, after epoch 140, the loss became very big(G_loss changed from 2 to 800，currently,It's72341936 at 170 epoch), and unable to convergence. I just dont know why
